layouts/page/superpowers.html

```diff
@@ -0,0 +1,276 @@
+{{ define "hero" }}
+    <header class="header-hero shadow-2xl relative overflow-hidden">
+        <div class="absolute hero-clouds w-full"><img src="/images/hero-clouds.svg" alt=""></div>
+        <div class="container mx-auto">
+            <div class="flex items-start">
+                <div>
+                    <img class="w-64" src="/images/mascot/pulumipus.svg" alt="">
+                </div>
+                <div class="md:ml-8 md:mt-8 md:w-3/4">
+                    <h1 class="text-4xl">Modern Infrastructure with <span class="text-blue-300 font-semibold">Superpowers</span>.</h1>
+                    <p class="text-xl pr-48 text-purple-100 leading-relaxed">
+                        We're unleashing new capabilities for Delivery, Provisioning,
```

Comment:
```suggestion
                        Pulumi 2.0 includes new capabilities for Delivery, Provisioning,
```

One thing to keep in mind is that the features are available as soon as this page goes live. We don't need to make it look like they're somehow held back until after the webinar.